smile_id_core/Signature.py

```diff
@@ -17,6 +17,7 @@ def __init__(self, partner_id: str, api_key: str):
         self.partner_id = partner_id
         self.api_key = api_key
 
+    @deprecated("since version 2.0.0")
```

Comment:
You leave the methods in, but remove the tests. If we are going to leave them in, then the tests should stay in too. Also, you forgot to include the deprecation library as a requirement, which is why your tests are failing.

Reply:
That's it! Everything that "smells" `sec_key` must go! 😉